### ISSUE TITLE
This common branch can be used for Realme GT 2 master explorer edition?

### ISSUE THREAD

**Issue by Elkowak** [pjgowtham/android_device_oplus_sm8450-common#1]
Hi there, i have a realme gt2 master explorer device, it's similar to gt2 pro but with snapdragon 8+ gen 1 but, it a chinese only device and chinese realmeui sucks, i didn't see any developer working on custom rom aosp based for this device, only custom realme ui on 4pda..

As this device has a lot of similarity in firmware with the GT2 Pro and OnePlus 11R, is it possible to use this common branch for build lineage or even AOSP?

this device already have a firmware dump by tadiphone: https://dumps.tadiphone.dev/dumps/realme/re5465/tree/qssi-user-14-UKQ1.230924.001-1703251834605-release-keys--ALLNET/

**Comment by pjgowtham** [comment]
Yes, it can be used to build for any oneplus/realme/oppo - 8 gen 1 or 8+ gen 1

**Comment by Elkowak** [comment]
I just tried but no success, device refuse to boot.. even with prebuilt kernel, maybe something is wrong on kernel source or device tree.. but i dont have enough time to debug, btw thanks for your effort on oneplus/realme/oppo - sm8450/8475

**Comment by pjgowtham** [comment]
Use prebuilt DTB/DTBO. Can it boot recovery ?

**Comment by Elkowak** [comment]
> Use prebuilt DTB/DTBO. Can it boot recovery ?
> 

I used prebuilt DTB/DTBO and source built after adding missing device tree files.

I can only boot to recovery with stock boot.img and vendor_boot.img
